pkg/policyexception/policyexception.go

```diff
@@ -0,0 +1,16 @@
+package policyexception
+
+import (
+	kyvernov1 "github.com/kyverno/kyverno/api/kyverno/v1"
+)
+
+type ExcludeResource []kyvernov1.MatchResources
+
+// We need to create objects for this interface so that
+// resourceHandler can easily call the method to get policy exceptions
+type Exceptions interface {
+	// Get corresponding rules and exceptions by policy
+	// Return `ExcludeResource` which is an array, including elements:
+	// `kyvernov1.MatchResources`
+	GetExceptionByPolicyRule(policy kyvernov1.PolicyInterface, ruleName string) ExcludeResource
```

Comment:
Rename to `GetExceptionsByPolicyRule`.

In Go naming it may be OK to also drop the `Get` and simply name as `ExceptionsByRule`.

Reply:
Sounds good! Just simplified the name to `ExceptionsByRule`